app/models/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+class Authorization
+  attr_accessor :token
+  attr_reader :token_auth_accounts
+
+  def initialize(token)
+    @token = token
+  end
+
+  def id
```

Comment:
strangely, I think it's the `to_json` method. Looks like ActiveRecord has its own `to_json`, and I think since Authorizations are not db objects and therefore have no IDs, it's erroring out on `let(:json) { JSON.parse(representer.to_json) }`